.github/workflows/e2e-vind-smoke.yml

```
@@ -14,6 +14,7 @@ concurrency:
 jobs:
   smoke-test:
     name: create->kubectl->delete
+    if: false  # Temporarily disabled for GA merge surge
     runs-on: ubuntu-latest
     if: vars.SUMMIT_VIND_CI == '1'
```

YAMLlint (1.38.0)

[error] duplication of key "if" in mapping

(key-duplicates)